Content.Client/HealthAnalyzer/UI/HealthAnalyzerWindow.xaml.cs

```diff
@@ -67,7 +67,7 @@ public void Populate(HealthAnalyzerScannedUserMessage msg)
                             if (!shownTypes.Contains(type))
                             {
                                 shownTypes.Add(type);
-                                text.Append($"\n- {Loc.GetString("health-analyzer-window-damage-type-text", ("damageType", type), ("amount", typeAmount))}");
+                                text.Append($"\n- {Loc.GetString("health-analyzer-window-damage-type-text", ("damageType", Loc.GetString("health-analyzer-window-damage-type-" + type.ToLower())), ("amount", typeAmount))}");
```

Comment:
instead of doing `ToLower()` here and above, just have the ID in ftl also be uppercase at the end--we agreed on this mixed case on occasion for other localized stuff, and it makes it obvious that its an 'auto-generated' one

Reply:
Fixed.